bindings/pyroot_experimental/PyROOT/src/TDirectoryPyz.cxx

```diff
@@ -0,0 +1,100 @@
+// Author: Danilo Piparo, Stefan Wunsch CERN  08/2018
+// Original PyROOT code by Wim Lavrijsen, LBL
+
+/*************************************************************************
+ * Copyright (C) 1995-2018, Rene Brun and Fons Rademakers.               *
+ * All rights reserved.                                                  *
+ *                                                                       *
+ * For the licensing terms see $ROOTSYS/LICENSE.                         *
+ * For the list of contributors see $ROOTSYS/README/CREDITS.             *
+ *************************************************************************/
+
+#include "CPyCppyy.h"
+#include "CPPInstance.h"
+#include "CallContext.h"
+#include "PyROOTPythonize.h"
+#include "TPython.h"
+#include "Utility.h"
+
+#include "PyzCppHelpers.hxx"
+
+#include "TClass.h"
+#include "TDirectory.h"
+
+#include "Python.h"
+
+using namespace CPyCppyy;
+
+PyObject *TDirectoryGetObject(CPPInstance *self, PyObject *args)
```

Comment:
Could you add some documentation here about what this pythonization actually does?

Reply:
Same comment applies to the other pythonizor methods you implemented.

Reply:
To be used as reference, [this](https://github.com/root-project/root/blob/360b252c6cbab44b5e742bddc236500992d8d15d/bindings/pyroot_experimental/PyROOT/src/TTreePyz.cxx) for `AddBranchAttrSyntax` and `SetBranchAddressPyz`.

Reply:
Sure. I support the more expressive way of re-writing pythonisations.

Reply:
Implemented.